packaging/pkcs11-provider.spec

```diff
@@ -42,7 +42,6 @@ autoreconf -fi
 %install
 %make_install
 mkdir %{buildroot}%{_libdir}/ossl-modules/
-ln -s ../pkcs11_provider.so %{buildroot}%{_libdir}/ossl-modules/pkcs11.so
```

Comment:
I'm not that familiar with spec files, but is the `mkdir` line still necessary? I would expect, that the previous `%make_install` already created the ossl-modules directory. Although it won't hurt, I would prefer to remove it.

Reply:
@jakuje Given I need to fix the PR I guess I can add a job to build the RPM after distcheck generates the tarball, right ?

Reply:
There are different ways but certainly it should go into separate PR.

The packit usually just makes tarball from the currently committed files and creates SRPM from that and can build that either locally or in copr. AFAIK there is no way to pass it the tarball from make dist. 

Default integration requires installing Github application to the organization: https://packit.dev/docs/guide/#via-fedora-copr but for opensc we went with simple container running the packit and building the package locally: https://github.com/OpenSC/OpenSC/pull/2625/ (and https://github.com/OpenSC/OpenSC/pull/2528 for the packit configuration).

Reply:
I just added a rpmbuild step to this PR.
It is just a small change, I do not think we need a whole new PR
It may fail in CI because I probably ned to pull in some more packages to have rpmbuild work, but you can look at the distcheck job change and let me know what you think.

Reply:
Right. Packit does not do much more fancy sftuff than calling rpmbuild internally so using the rpmbuild like this is totally ok.